.gitignore

```diff
@@ -1,3 +1,4 @@
+*~
```

Comment:
What is this ignore for?

Reply:
emacs back-up files. Last saved version of any edited file.